satellite/audit/reservoirchore.go

```diff
@@ -0,0 +1,85 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit
+
+import (
+	"context"
+	"math/rand"
+	"time"
+
+	"go.uber.org/zap"
+
+	"storj.io/storj/internal/sync2"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/satellite/metainfo"
+)
+
+// ReservoirChore populates reservoirs and the audit queue.
+type ReservoirChore struct {
```

Comment:
If the other audit thing is going away, this could be named just `Chore`, because there wouldn't be a naming conflict.